src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -76,10 +76,14 @@ public static TestSuite suite() throws IOException {
       if (Arrays.asList("partial", "multifilePartial", "partialCrossModuleTypeImports")
           .contains(input.getParentFile().getName())) {
         subject.partialInput = true;
+        subject.debug = false;
       }
       if (input.getParentFile().getName().equals("partialCrossModuleTypeImports")) {
         subject.depgraph = "partialCrossModuleTypeImports/cross_module_type.depgraph";
       }
+      if (input.getName().contains("async")) {
```

Comment:
Comment why async is special. Its ok if we don't fully understand the error and the comment is "errors mysterously", but needs to have one.

Reply:
Done.